lxd/instance/drivers/driver_lxc.go

```diff
@@ -1713,6 +1713,31 @@ func (d *lxc) deviceHandleMounts(mounts []deviceConfig.MountEntryItem) error {
 				}
 			}
 
+			// If the mount is a directory, create the LXD marker file.
+			if shared.IsDir(mount.DevPath) {
+				// Connect to files API.
+				files, err := d.FileSFTP()
+				if err != nil {
+					return err
+				}
+
+				defer func() { _ = files.Close() }()
+
+				_, err = files.Lstat(mount.TargetPath)
+				if err == nil {
+					// The target path already exist.
+					err = d.deviceVolatileSetFunc(mount.DevName)(map[string]string{fmt.Sprintf("%s.dir_created_by_lxd", mount.TargetPath): "false"})
```

Comment:
Lets use `%s.last_state.created` so we align with the NIC volatile keys that indicate something similar

Reply:
Lets have a constant for this too so we can see where its referred to.

Reply:
fixed